src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+public class Ui {
+    
+    private static String LINES = "--------------------------------------------------------------------------------------\n";
+    
+    public static void intro() {
+        String logo = " ____        _        \n"
```

Comment:
Would it help with the legibility of the code if the logos and messages were to be made into static constants?